.gitignore

```diff
@@ -1,2 +1 @@
 .DS_Store
-
```

Comment:
idk this change seems pretty reckless right here

Reply:
Lmao there were two blank lines in that file for some reason, so I removed one.